helm/cluster-eks/values.schema.json

```
@@ -396,6 +396,31 @@
                             ],
                             "default": "public"
                         },
+                        "encryptionConfig": {
+                            "type": "object",
+                            "title": "Encryption config",
+                            "description": "Encryption configuration for the Kubernetes API server.",
+                            "properties": {
+                                "keyARN": {
```

```suggestion
                                "keyArn": {
```

Since we also use non-capital camelCase elsewhere, such as cluster-aws (`global.connectivity.subnets[*].natGatewayId`, `global.controlPlane.loadBalancerIngressAllowCidrBlocks`, ...).